web/src/App.test.jsx

```diff
@@ -62,6 +62,7 @@ jest.mock("~/queries/l10n", () => ({
 jest.mock("~/queries/issues", () => ({
   ...jest.requireActual("~/queries/issues"),
   useIssuesChanges: () => jest.fn(),
+  useAllIssues: () => ({ isEmtpy: true }),
```

Comment:
:warning: :warning: :warning: 

After moving the `core/InstallButton` to the top bar and using the `useAllIssues` to decide if it should be shown (aka mounted) or not, I have had to add this mock here and in the _components/core/Header.test.tsx_ file to make these tests work again.

This made me ask myself if would be a good idea to have some test util for always have issues mocked and influence them when needed.